Casks/i/insomnia.rb

```diff
@@ -9,7 +9,7 @@
   homepage "https://insomnia.rest/"
 
   livecheck do
-    regex(/core@v?(\d+(?:\.\d+)+)(?!.)/i)
+    regex(/core@v?(\d{1,3}(?:\.\d+)+)(?!.)/i)
```

Comment:
An alternative to consider could be to explicitly negate the `20##` versions.
```suggestion
    regex(/core@v?((?!20\d+)\d+?(?:\.\d+)+)(?!.)/i)
```

Reply:
I'm good with avoiding the date-based versions by restricting the number of digits (that's how I typically handle this particular situation). I'll push some changes in a moment to add explanatory comments and tweak the regexes a little.